### PR TITLE
Use new COS

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ FROM codait/max-base:v1.1.3
 RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
 
 # Fill in these with a link to the bucket containing the model and the model file name
-ARG model_bucket=https://max.cdn.appdomain.cloud/max-text-summarizer/1.0.0
+ARG model_bucket=https://max-cdn.appdomain.cloud/max-text-summarizer/1.0.0
```
Comment (bdwyer2) [inline Dockerfile]:
```suggestion
ARG model_bucket=https://max-cdn.cdn.appdomain.cloud/max-text-summarizer/1.0.0
```

**Review by xuhdev — APPROVED** [review]
Verified the link works

File: Dockerfile (hunk 1 above)
Comment (xuhdev) [reply Dockerfile]: Hmm... both links work?

Comment (xuhdev) [reply Dockerfile]: OK, turns out that I have only examined the new link. It works perfectly.

**Pull request merged**